examples/dt/cloudops/openstack-telemetry_DT.md

```diff
@@ -0,0 +1,56 @@
+# RHOSO Deployed Topology OpenStack Observability
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/78b3c876eaf9168f9d95b201997ebdc2da42fa02) on Oct 17th, 2023**
+
+## General information
+
+| Revision | Change                | Date             |
+|--------: | :-------------------- | :--------------: |
+| v0.1     | Initial publication   | 14.12.23      |
+
+## Node topology
+| Node role                                     | bm/vm | amount |
+| --------------------------------------------- | ----- | ------ |
+| Compute nodes                                 | vm    | 2      |
+| Controller nodes                              | vm    | 3      |
+
+
+## Services, enabled features and configurations
+| Service                                     | configuration                     | Lock-in coverage?  |
+| ------------------------------------------- | ----------------------------------| ------------------ |
+| Nova                                        | default                           | Standard           |
+| Neutron                                     | default                           | Standard           |
+| Cinder                                      | default                           | Standard           |
+| Glance                                      | default                           | Standard           |
+| Keystone                                    | default                           | Standard           |
+| Memcached                                   | default                           | Standard           |
+| RabbitMQ                                    | default                           | Standard           |
+| Telemetry                                   | aka ceilometer, aodh & prometheus | Must have          |
+
+## Considerations/Constraints
+1. Telemetry should be enabled that will enable autoscaling, telemetry, prometheus deployment, grafana deployment, etc.
+2. No topology constraints
+
+
+## Testing tree
+
+| Test framework       | Stage to run  | Special configuration | Test case to report |
+| -------------------- | ------------  | --------------------- | :-----------------: |
+| Tempest-telemetry    | stage 10      | Use rhel image        |    TBD         |
+| Autscaling           | stage 11      | Use cirros image      |    TBD         |
```

Comment:
```suggestion
| Auto scaling         | stage 11      | Use cirros image      |    TBD         |
```

Reply:
It should be Autoscaling, without space. I have fixed it.